src/Between.php

```diff
@@ -164,6 +164,10 @@ public function isValid($value)
     {
         $this->setValue($value);
 
+        if ($this->getMin() === null || $this->getMax() === null) {
+            throw new Exception\InvalidArgumentException("Missing option. 'min' and 'max' have to be given");
```

Comment:
As explained in the reply I just made to @Geeh, it's not to let you instantiate a validator and keep it in an invalid state, but instead to allow you to avoid using the "array of options with arbitrary string keys" if you want to.

Related issue I opened https://github.com/zendframework/zend-validator/issues/16 - the current tests show an example of the Between validator being instantiated with invalid options, but not throwing an exception (because there are 2 values in the array), and the validator is then giving false positives because t falls back to the default values (0 and PHP_INT_MAX). This PR works around that, because if you pass in invalid options to the constructor (leaving the validator in an invalid state), the isValid function will throw the exception